config/environments/production.rb

```diff
@@ -61,6 +58,11 @@
   # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
   # config.assets.precompile += %w( search.js )
 
+  # Use a real queuing backend for Active Job (and separate queues per environment)
+  # config.active_job.queue_adapter     = :resque
+  # config.active_job.queue_name_prefix = "railsdiff_#{Rails.env}"
```

Comment:
`railsdiff` was probably their app name, if anyone cares this comment can be changed to `samson`